tests/integration/cli/action/test_action_classification.py

```diff
@@ -0,0 +1,58 @@
+"""Tests for MPA Class-Incremental Learning for object detection with OTX CLI"""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from otx.api.entities.model_template import parse_model_template
+from otx.cli.registry import Registry
+from otx.cli.utils.tests import otx_eval_testing, otx_train_testing
+from tests.test_suite.e2e_test_system import e2e_pytest_component
+
+# Finetuning arguments
+args = {
+    "--train-ann-file": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
+    "--train-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
+    "--val-ann-file": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
+    "--val-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
+    "--test-ann-files": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
+    "--test-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
+    "train_params": ["params", "--learning_parameters.num_iters", "2", "--learning_parameters.batch_size", "4"],
+}
+
+otx_dir = os.getcwd()
+
+TT_STABILITY_TESTS = os.environ.get("TT_STABILITY_TESTS", False)
+if TT_STABILITY_TESTS:
+    default_template = parse_model_template(
+        os.path.join("otx/algorithms/action/configs", "classification", "x3d", "template.yaml")
+    )
+    templates = [default_template] * 100
+    templates_ids = [template.model_template_id + f"-{i+1}" for i, template in enumerate(templates)]
+else:
+    templates = Registry("otx/algorithms/action").filter(task_type="ACTION_CLASSIFICATION").templates
```

Comment:
proper indentation is needed

Reply:
I cannot see any indentation error. Could you write more specifically?

Reply:
Ah, sorry, I mis-looked